server/src/main/java/org/apache/druid/server/lookup/cache/LookupCoordinatorManager.java

```diff
@@ -557,16 +557,16 @@ void lookupManagementLoop()
 
     try {
       List<ListenableFuture<Map.Entry>> futures = new ArrayList<>();
-      for (Map.Entry<String, Map<String, LookupExtractorFactoryMapContainer>> tierEntry : allLookupTiers.entrySet()) {
+      for (String tier : lookupNodeDiscovery.getAllTiers()) {
```

Comment:
can you merge https://github.com/FaxianZhao/incubator-druid/pull/1 to add another block to note lookups that will never be loaded?

Reply:
> can you merge [FaxianZhao#1](https://github.com/FaxianZhao/incubator-druid/pull/1) to add another block to note lookups that will never be loaded?

Good idea.
But for the users who enable 'druid.lookup.lookupTierIsDatasource', will this warning confuse them?

Reply:
a quick look at the usage of that property tells me that it is not used by coordinator and shouldn't interfere.

Reply:
> a quick look at the usage of that property tells me that it is not used by coordinator and shouldn't interfere.

Agree.